docker/Dockerfile-tao-release

```diff
@@ -1,7 +1,7 @@
 #
 # Dockerfile used to generate https://hub.docker.com/repository/docker/taotesting/tao-release
 #
-FROM ubuntu:18.04
+FROM ubuntu:20.04
```

Comment:
Sounds good since we are using ubuntu 20.04 to run our pipelines and many EC2 on production